packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -926,13 +926,13 @@ class EditableText extends StatefulWidget {
   /// and selection, one can add a listener to its [controller] with
   /// [TextEditingController.addListener].
   ///
-  /// {@tool dartpad --template=stateful_widget_material_no_null_safety}
+  /// {@tool dartpad --template=stateful_widget_material}
   ///
   /// This example shows how onChanged could be used to check the TextField's
   /// current value each time the user inserts or deletes a character.
   ///
   /// ```dart
-  /// TextEditingController _controller;
+  /// late TextEditingController _controller;
```

Comment:
I think this could just be: 
```suggestion
  /// final TextEditingController _controller = TextEditingController();
```

And remove the `initState` override

Reply:
Yeah, I thought about that. I kind of like having it constructed in the `initState` to mirror disposing it in `dispose` as it is sometime hard to remember you need to dispose it. That said for a small example like this it might be better to be more concise.